exchanges/order/orders.go

```diff
@@ -662,8 +662,12 @@ func StringToOrderStatus(status string) (Status, error) {
 		return PartiallyCancelled, nil
 	case strings.EqualFold(status, Open.String()):
 		return Open, nil
+	case strings.EqualFold(status, Closed.String()):
+		return Closed, nil
 	case strings.EqualFold(status, Cancelled.String()):
 		return Cancelled, nil
+	case strings.EqualFold(status, Canceled.String()): // Kraken case
```

Comment:
Thanks for adding the new case to `order.StringToOrderType()`! However, to prevent issues of someone actually using "Canceled" as an order type accidentally, could you make `Canceled.String()` a string and remove the `Canceled` `order.Type`?

Reply:
+